README.md

```diff
@@ -1,144 +1,86 @@
-
-__Note:__ We strongly recommend to always use the latest version of the proxy,
-and to update the version regularly. However, we recommend pinning to a
-specific tag and avoid the `latest` tag. Additionally, please note that
-the tagged version is _only_ that of the proxy - changes in base images may 
-break specific setups, even on non-major version increments. As such,
-it's a best practice to test changes before deployment, and use automated
-rollbacks to revert potential failures. 
-
-## To use from Kubernetes:
-
-### Deploying Cloud SQL Proxy as a sidecar container
-Follow this [page](https://github.com/GoogleCloudPlatform/cloudsql-proxy/tree/main/examples/kubernetes#run-the-cloud-sql-proxy-as-a-sidecar). See also
-[Connecting from Google Kubernetes Engine](https://cloud.google.com/sql/docs/mysql/connect-kubernetes-engine).
+### Installation
 
+The proxy may be installed as a [gcloud component][gcloud-install], or as a
+[separate binary][binary-install]. See [Releases][releases] for all past
+versions.
 
-## Third Party
+In addition, there are containerized versions of the proxy available from the
+following Google Cloud Container Registry repositories:
 
-__WARNING__: _These distributions are not officially supported by Google._
+* `gcr.io/cloudsql-docker/gce-proxy`
+* `us.gcr.io/cloudsql-docker/gce-proxy`
+* `eu.gcr.io/cloudsql-docker/gce-proxy`
+* `asia.gcr.io/cloudsql-docker/gce-proxy`
 
-### Installing via Homebrew
+Each image is tagged with the associated proxy version. The following tags are
+currently supported:
 
-  You can find a formula for with Homebrew [here](https://github.com/tclass/homebrew-cloud_sql_proxy).
+* `$VERSION` - default image (recommended)
+* `$VERSION-alpine` - uses [`alpine:3`](https://hub.docker.com/_/alpine)
+  as a base image (only supported from v1.17 up)
+* `$VERSION-buster` - uses [`debian:buster`](https://hub.docker.com/_/debian)
+  as a base image (only supported from v1.17 up)
 
+### Command Line Invocation
 
-### K8s Cluster Service using Helm
+All the following invocations assume valid credentials are present in the
+environment. For more thorough documentation on the support command line flags,
+see the [CLI usage docs](docs/cli-usage.md). For more information on using the
+proxy as a library, see the [associated docs](docs/library-usage.md).
 
-  Follow these [instructions](https://github.com/rimusz/charts/tree/master/stable/gcloud-sqlproxy).
-  This chart creates a Deployment and a Service, but we recommend deploying the proxy as a sidecar container in your pods.
+#### Unix socket example
 
-### .Net Proxy Wrapper (Nuget Package)
-  
-  Install via Nuget, follow these [instructions](https://github.com/expert1-pty-ltd/cloudsql-proxy#install-via-nuget).
+``` bash
+# Must create a /cloudsql directory first
+cloud_sql_proxy -dir=/cloudsql -instances=my-project:us-central1:my-instance
+```
+
+#### TCP socket example
```

Comment:
Please move TCP to the top - going forward, it should be messaged as the default path.